lib/MainPageHelpers/personalPlanWidget.dart

```diff
@@ -263,65 +108,56 @@ class _PersonalPlanWidgetState extends State<PersonalPlanWidget> {
-                    );
-//TODO: think of a way to do the other option to chose where to download the file to (like the share option)
-                    showDialog(
-                      // the dialog that shows the user that the download has started and finished
-                      context: context,
-                      builder: (BuildContext context) {
-                        return AlertDialog(
-                          title: Directionality(
-                            textDirection: TextDirection.rtl,
-                            child: Text(appInfoProvider
-                                    .returnToPlanStrings['StartedDownload'] ??
-                                'ההורדה החלה'),
-                          ),
-                          content: Directionality(
-                            textDirection: TextDirection.rtl,
-                            child: Text(appInfoProvider
-                                    .returnToPlanStrings['FinishDownload'] ??
-                                'התוכנית ירדה'),
-                          ),
-                          actions: <Widget>[
-                            TextButton(
-                              child: Directionality(
-                                textDirection: TextDirection.rtl,
-                                child: Text(appInfoProvider
-                                        .returnToPlanStrings['Finish'] ??
-                                    'סיום'),
-                              ),
-                              onPressed: () {
-                                Navigator.of(context).pop();
-                              },
-                            ),
-                          ],
-                        );
-                      },
-                    );
+                    var result = await fileService.download([
+                      appInfoProvider
+                          .formDifficultEventsTitles['header$gender'],
+                      appInfoProvider.formMakeSaferTitles['header$gender'],
+                      appInfoProvider.formFeelBetterTitles['header$gender'],
+                      appInfoProvider.formDistractionsTitles['header$gender'],
+                      appInfoProvider.formPhonePage['header$gender'],
+                    ], [
+                      appInfoProvider
+                          .formDifficultEventsTitles['subTitle$gender'],
+                      appInfoProvider.formMakeSaferTitles['subTitle$gender'],
+                      appInfoProvider.formFeelBetterTitles['subTitle$gender'],
+                      appInfoProvider.formDistractionsTitles['subTitle$gender'],
+                      appInfoProvider.formPhonePage['subTitle$gender'],
+                    ], appInfoProvider.sharePDFtexts, ShareFileType.PDF);
+                    if (result == null) {
+                      // Show him a message
+                      showToast(message: 'ההורדה נכשלה');
+                      return;
```

Comment:
Isn't this a failure we want to know about? if yes, let's send it to sentry

Reply:
I didn't add any sentry to this pr. wouldn't it be better to have a seperate branch/pr that deals with integrating it?

Reply:
IMHO error handling are integral part of development flow so I expect it to be together with the code.

Reply:
Isn't it better for future development to have sentry implementation (i.e sending the data to sentry) as a seperate pr? such that in the event that we want to remove it, we have a good way of knowing exactly where we called it?